apps/src/blockTooltips/DropletFunctionTooltip.js

```diff
@@ -67,14 +68,15 @@ var DropletFunctionTooltip = function (appMsg, definition) {
   /** @type {Array.<parameterInfo>} */
   this.parameterInfos = [];
 
-  var paramId = 0;
-  var paramName;
-  while (!!(paramName = appMsg[this.parameterNameKey(paramId)] ||
-                          msg[this.parameterNameKey(paramId)])) {
+  for (var paramId = 0; ; paramId++) {
```

Comment:
Not super happy with the for-loop without a condition.  Open to suggestions for how to better write this.